metaflow/plugins/pypi/conda_environment.py

```diff
@@ -282,6 +282,11 @@ def get_environment(self, step):
             # Match PyPI and Conda python versions with the resolved environment Python.
             environment["pypi"]["python"] = environment["conda"]["python"] = env_python
 
+            if os.getenv("GOOGLE_APPLICATION_CREDENTIALS"):
```

Comment:
can you add a comment in the code why this is desired

Reply:
done